lib/user.rb

```diff
@@ -2,12 +2,28 @@
 require 'dm-validations'
 require 'dm-postgres-adapter'
```

Comment:
Similar to peep.rb, remove references to gems and put them in your gemfile (if they're not there already). Also retain require 'data_mapper'